docs/framework/additional-apis/toc.yml

```diff
@@ -82,8 +82,28 @@
       href: datamemberfieldeditor-class.md
     - name: DataMemberListEditor class
       href: datamemberlisteditor-class.md
+  - name: adodb Namespace
+    items:
+    - name: Connection Interface
+      href: adodb.connection.md
+    - name: EventReason Enum
+      href: adodb.eventreasonenum.md
+    - name: EventStatus Enum
+      href: adodb.eventstatusenum.md
   - name: stdole Namespace
     items:
+    - name: DISPPARAMS Structure
+      href: stdole.dispparams.md
+    - name: EXCEPINFO Structure
+      href: stdole.excepinfo.md
+    - name: IFontDisp Interface
+      href: stdole.ifontdisp.md
+    - name: IPictureDisp Interface
+      href: stdole.ipicturedisp.md
+    - name: StdFont Interface
```

Comment:
Can you add these in alphabetical order?

Reply:
@mairaw  Sure, I thought it was better to be by type (i.e. Structures, Intefaces, Properties) and alphabetical within each type

Reply:
I've sorted the stdole entries alphabetical